benchalerts/tests/unit_tests/mocks.py

```diff
@@ -85,11 +105,20 @@ def send(self, *args, **kwargs):
 
 
 class MockConbenchClient(ConbenchClient):
+    """A ConbenchClient that uses MockAdapter to intercept all requests and return
+    mocked responses, constructed from JSON files.
+    """
+
     def __init__(self):
         super().__init__()
         self.session.mount("https://", MockAdapter())
 
     def _login_or_raise(self) -> None:
+        # Since we mock all requests, logging in won't change any state, so let's skip
+        # logging in. Logging in should be tested in the benchclients library already.
+        #
+        # (Also, as of this comment, the parent method sets up a new requests session,
+        # which would mean we'd lose our MockAdapter.)
```

Comment:
Just curious: Is there a reason this is a comment and not a docstring? Are we opposed to docstrings for internal methods?

Reply:
I was thinking this was more of a code comment about the next line, `pass`, than a description of this method. But either way would work.